tests/thread/stress_create.py

```diff
@@ -15,7 +15,8 @@ def thread_entry(n):
         _thread.start_new_thread(thread_entry, (thread_num,))
         thread_num += 1
     except MemoryError:
-        pass
+        # let other threads run
+        time.sleep(0.05)
```

Comment:
Ummm, I have tried in several different ways, and without the time.sleep() line, it always gets stuck raising the memory error interrupt.